src/shared/debug/launchConfiguration.ts

```diff
@@ -37,7 +41,14 @@ export class LaunchConfiguration {
     ) {}
 
     public getDebugConfigurations(): vscode.DebugConfiguration[] {
-        return this.configSource.getDebugConfigurations()
+        return _(this.configSource.getDebugConfigurations())
+            .map(o => {
+                if (isAwsSamDebugConfiguration(o)) {
+                    ensureRelativePaths(undefined, o)
+                }
+                return o
```

Comment:
@ckant any idea how `getDebugConfigurations()` would be returning absolute paths here? It's not hitting `provideDebugConfigurations()` nor any other codepath of ours AFAICT.

Reply:
It's not clear to me why it would be related to this particular line (*get*). 

I makes sense to me that it would *create* debug configs with absolute paths since it uses `document.uri` to determine the template uri.

https://github.com/aws/aws-toolkit-vscode/blob/feature/debugconfig/src/shared/codelens/samTemplateCodeLensProvider.ts#L38